backend/tests/clubs/test_ticketing.py

```diff
@@ -115,6 +121,7 @@ def test_create_ticket_offerings_bad_perms(self):
         self.client.login(username=self.user2.username, password="test")
         qts = {
             "quantities": [
+                {"type": "_free", "count": 30, "price": 0},
```

Comment:
Don't need to add this here. Your changes are isolated to `initiate_checkout`. Same goes for all the other changes to tests.

Reply:
Don't you think if we're going to have free tickets we still want to test how they behave in other instances where the user buys free and non-free tickets to catch bugs we wouldn't otherwise see if free tickets were left out?

Reply:
Good point -- supporting free tickets means that they should work end-to-end, even if the only code change is in `initiate_checkout`. I still think adding them globally (and integrating them with all current tests) is a bit much. Maybe we can add specific tests (for only free tickets) for all the major routes? i.e. `test_create_ticket_offerings_free_tickets`

Reply:
Even if this means calling `Ticket.objects.create(type="free", ...` every time we want to test free tickets?

Reply:
That'll get called at each test regardless, whether it's in the setup or in an individual test case. ([`setUp` gets called between each case.](https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUp)) 

For the sake of getting this PR shipped, let's leave the creation in setup for now -- or whatever impl you currently have, provided it works.

Reply:
It's ok, I can make the changes right now